catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -924,15 +906,15 @@ public static void printNode(Node node, int offset) {
 
     if (node.node_type == NodeType.ATTRIBUTE_NODE) {
       if (node.value != null && node.value.type() != null) {
-        value = CorbaUtils.getNodeValue(node.value);
+        String value = CorbaUtils.getNodeValue(node.value);
         sb.append(attrName);
         sb.append("=");
         sb.append(value);
```

Comment:
could just do `sb.append(CorbaUtils.getNodeValue(node.value))` and not have the local variable